scripts/build/electron-installer-debian-linux.sh

```diff
@@ -62,10 +62,15 @@ then
   usage
 fi
 
+if [ ! -d "$PWD/node_modules" ]; then
```

Comment:
This should probably be `"./node_modules"` to match the usage on line 73? (and line 28)